packages/core/src/builder/buildProject.ts

```diff
@@ -7,13 +7,15 @@ import { Plugin } from "../cli";
 
 export interface BuildCLIOptions {
   revision?: string;
+  schemaVersion?: string;
 
   // all three together
   environment?: string;
   feature?: string;
   print?: boolean;
   pretty?: boolean;
   stateFiles?: boolean; // --no-state-files in CLI
+  inflate?: number;
```

Comment:
when building/testing/benchmarking, you can pass `--inflate=100` in CLI, which will make the generated datafile size go up by 100 times.

this is meant to be an internal API for testing and benchmarking purposes, which end users don't necessarily have to know about.

```
$ npx featurevisor build --inflate=100
```